apps/dashboard/app/(app)/audit/[bucket]/filter.tsx

```
@@ -72,41 +72,36 @@ export const Filter: React.FC<Props> = ({ options, title, param }) => {
-                    key={option.value}
-                    onSelect={() => {
+                  <div
+                    onMouseDownCapture={() => {
                       const next = isSelected
                         ? selected.filter((v) => v !== option.value)
                         : Array.from(new Set([...selected, option.value]));
-
                       setSelected(next);
-                      // params.set(param, next);
                     }}
                   >
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)